numba/tests/doc_examples/test_interval_example.py

```diff
@@ -0,0 +1,187 @@
+"""
+This test is used by `docs/source/extending/interval-example.rst`.
+
+The "magictoken" comments are used as markers for the beginning and ending of example code.
```

Comment:
Sphinx is complaing here:

numba/tests/doc_examples/test_interval_example.py:4:81: E501 line too long (91 > 80 characters)

```suggestion
The "magictoken" comments are used as markers for the beginning and ending of
example code.
```